### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by Max0293**

### CODE REVIEW

File: homework/mkra/index.js [hunk 1]
```diff
@@ -1 +1,46 @@
...
+    	console.log('Fizz');
+    }
+	else {
+	console.log(counter);
+    }
+counter++;
+}
+// Task 2
+function isPall(message){
+    if (message === null || message === undefined ||
```
Comment (kovm) [inline homework/mkra/index.js]: your condition is over complicated just use !message

File: homework/mkra/index.js [hunk 2]
```diff
@@ -1 +1,46 @@
...
+counter++;
+}
+// Task 2
+function isPall(message){
+    if (message === null || message === undefined || 
+        message === '' || message ===' '){
+            console.log('Incorrect data');
+            return;
+    }
+    let msg = String(message);
```
Comment (kovm) [inline homework/mkra/index.js]: Such type coercion is prohibited 
Use **typeof** operator

Comment (kovm) [reply homework/mkra/index.js]: e.g. your solution is still working for arrays
isPall([1,2,1]) returns true

File: homework/mkra/index.js [hunk 3]
```diff
@@ -1 +1,46 @@
...
+    let result = '';
+    let arr = [];
+    let counter = 0;
+    for (let i = 0; i < msg.length; i++){
```
Comment (kovm) [inline homework/mkra/index.js]: Take a look to String.prototype.replace() method
e.g. you can replace all empty characters before Incorrect data check if it is possible )

Comment (Max0293) [reply homework/mkra/index.js]: I want to clarify:
array is considered as invalid data or i must check values?

Comment (kovm) [reply homework/mkra/index.js]: An array isn't valid data

File: homework/mkra/index.js (hunk 2 above)
Comment (Max0293) [reply homework/mkra/index.js]: All wishes are corrected, except for one. I don't understand how to compare a number without casting to String explicitly. If this is real, I will continue to reflect on it.

**Pull request merged**